config.json

```diff
@@ -572,7 +572,13 @@
       "core": false,
       "difficulty": 7,
       "slug": "word-search",
-      "topics": null,
+      "topics": [
+        "string",
```

Comment:
This should be `strings` to match the [topics list](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt). You can add language specific topics that aren't there if you feel the need, but otherwise we should remain consistent with that list as those topics will be used by the other language tracks :)